README.md

```diff
@@ -30,6 +30,36 @@ refget-compliance report -s https://refget.server.com/ --json server.json
 
 Setting `--json -` will have the compliance suite write the JSON to STDOUT.
 
+## Running the compliance suite on a docker container
+
+### Prerequisites
```

Comment:
Let's assume that the user doesn't have to create the docker image themselves, rather the image is available on the public registry and they can just `pull` it. So lines 37-50 can be removed.